src/translations/en.json

```diff
@@ -1503,12 +1503,12 @@
               "themes": "Themes"
             },
             "server_management": {
-              "heading": "Server management",
-              "introduction": "Control your Home Assistant.",
-              "restart": "Restart",
-              "confirm_restart": "Are you sure you want to restart Home Assistant?",
+              "heading": "Home Assistant Core",
```

Comment:
Is this more user friendly? I would argue this is more intimidating than just server management

Reply:
Home Assistant Core is the name of an installation method. This page is shown to all installation methods.